locale/update-i18n

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Just checking that this script assumes you've `bundle installed` from `foreman` and done an `npm install` inside `katello/engines/bastion_katello` ?